Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -104,46 +104,51 @@ theorem lintegral_nnnorm_neg {f : α → β} : (∫⁻ a, ‖(-f) a‖₊ ∂μ)
+/-- `HasFiniteIntegral f μ` means that the integral `∫⁻ a, ‖f a‖ ∂μ` is finite.
+  `HasFiniteIntegral f` means `HasFiniteIntegral f volume`. -/
+def HasFiniteIntegral {_ : MeasurableSpace α} (f : α → β) (μ : Measure α := by volume_tac) : Prop :=
   (∫⁻ a, ‖f a‖₊ ∂μ) < ∞
 #align measure_theory.has_finite_integral MeasureTheory.HasFiniteIntegral
 
+-- Porting note: TODO Delete this when leanprover/lean4#2243 is fixed.
+theorem hasFiniteIntegral_def {_ : MeasurableSpace α} (f : α → β) (μ : Measure α) :
+    HasFiniteIntegral f μ ↔ ((∫⁻ a, ‖f a‖₊ ∂μ) < ∞) :=
+  Iff.rfl
+
+attribute [eqns hasFiniteIntegral_def] HasFiniteIntegral
+
 theorem hasFiniteIntegral_iff_norm (f : α → β) :
     HasFiniteIntegral f μ ↔ (∫⁻ a, ENNReal.ofReal ‖f a‖ ∂μ) < ∞ := by
-  simp only [has_finite_integral, ofReal_norm_eq_coe_nnnorm]
+  simp only [HasFiniteIntegral, ofReal_norm_eq_coe_nnnorm]
 #align measure_theory.has_finite_integral_iff_norm MeasureTheory.hasFiniteIntegral_iff_norm
 
 theorem hasFiniteIntegral_iff_edist (f : α → β) :
     HasFiniteIntegral f μ ↔ (∫⁻ a, edist (f a) 0 ∂μ) < ∞ := by
-  simp only [has_finite_integral_iff_norm, edist_dist, dist_zero_right]
+  simp only [hasFiniteIntegral_iff_norm, edist_dist, dist_zero_right]
 #align measure_theory.has_finite_integral_iff_edist MeasureTheory.hasFiniteIntegral_iff_edist
 
 theorem hasFiniteIntegral_iff_ofReal {f : α → ℝ} (h : 0 ≤ᵐ[μ] f) :
     HasFiniteIntegral f μ ↔ (∫⁻ a, ENNReal.ofReal (f a) ∂μ) < ∞ := by
-  rw [has_finite_integral, lintegral_nnnorm_eq_of_ae_nonneg h]
+  rw [HasFiniteIntegral, lintegral_nnnorm_eq_of_ae_nonneg h]
 #align measure_theory.has_finite_integral_iff_of_real MeasureTheory.hasFiniteIntegral_iff_ofReal
 
-theorem hasFiniteIntegral_iff_of_nNReal {f : α → ℝ≥0} :
+theorem hasFiniteIntegral_iff_ofNNReal {f : α → ℝ≥0} :
     HasFiniteIntegral (fun x => (f x : ℝ)) μ ↔ (∫⁻ a, f a ∂μ) < ∞ := by
-  simp [has_finite_integral_iff_norm]
-#align measure_theory.has_finite_integral_iff_of_nnreal MeasureTheory.hasFiniteIntegral_iff_of_nNReal
+  simp [hasFiniteIntegral_iff_norm]
+#align measure_theory.has_finite_integral_iff_of_nnreal MeasureTheory.hasFiniteIntegral_iff_ofNNReal
 
 theorem HasFiniteIntegral.mono {f : α → β} {g : α → γ} (hg : HasFiniteIntegral g μ)
     (h : ∀ᵐ a ∂μ, ‖f a‖ ≤ ‖g a‖) : HasFiniteIntegral f μ := by
-  simp only [has_finite_integral_iff_norm] at *
+  simp only [hasFiniteIntegral_iff_norm] at *
   calc
     (∫⁻ a, ENNReal.ofReal ‖f a‖ ∂μ) ≤ ∫⁻ a : α, ENNReal.ofReal ‖g a‖ ∂μ :=
-      lintegral_mono_ae (h.mono fun a h => of_real_le_of_real h)
+      lintegral_mono_ae (h.mono fun a h => ofReal_le_ofReal h)
     _ < ∞ := hg
-    
 #align measure_theory.has_finite_integral.mono MeasureTheory.HasFiniteIntegral.mono
 
 theorem HasFiniteIntegral.mono' {f : α → β} {g : α → ℝ} (hg : HasFiniteIntegral g μ)
     (h : ∀ᵐ a ∂μ, ‖f a‖ ≤ g a) : HasFiniteIntegral f μ :=
-  hg.mono <| h.mono fun x hx => le_trans hx (le_abs_self _)
+  hg.mono <| h.mono fun _x hx => le_trans hx (le_abs_self _)
```

Comment:
I generally prefer to replace unused arguments just with `_` where possible, but it's not a big deal.

Reply:
I usually use `_` for unused arguments, but when the name of the unused argument is used in following hypotheses (like `hx` in this example), I use `_x` to make the name of hypotheses easy to understood.